client/listeners/index.js

```diff
@@ -221,6 +221,12 @@ function copyPath(el) {
         .catch(CloudCmd.log);
 }
 
+function execIfNotMobile(callback , event)
+{
+    if (window.innerWidth > CloudCmd.MOBILE_ONE_PANEL_WIDTH)
+        callback(event);
+}
+
```

Comment:
This is very good! Can we also use css variables here? Would be great!

Reply:
I thought we agreed trying it on different PR and replacing the other const with css one.

When this is merged i will try to switch both consts to css variable so the width value will be only defined on one place.